Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ArcadiaSparkBatchJob.java

```diff
@@ -78,6 +79,10 @@ public ArcadiaSparkBatchJob(final @NotNull SparkSubmissionParameter submissionPa
     @Override
     public Observable<SimpleImmutableEntry<MessageInfoType, String>> getSubmissionLog() {
         // No batches/{id}/log API support yet
+        getCtrlSubject().onNext(new SimpleImmutableEntry<>(
+                MessageInfoType.HtmlPersistentMessage,
```

Comment:
can we reuse the ctrlInfo method ?

Reply:
It's a different behavior and presentation.